.github/workflows/gh-lint.yaml

```
@@ -0,0 +1,152 @@
+        permissions:
+          contents: read
+          pull-requests: write
+        strategy:
+          matrix:
+            binary:
+              - actionlint
+              - ghalint
+        steps:
+          - name: Checkout
+            uses: actions/checkout@eef61447b9ff4aafe5dcd4e0bbf5d482be7e7871 # v4.2.1
+    
```

yamllint

[error] trailing spaces

(trailing-spaces)